bot/ts/ThreadSafeTSConnection.py

```diff
@@ -133,15 +141,18 @@ def ts3exec(self,
         return res, exres
 
     def close(self):
-        self.ts3exec(lambda tc: tc.close())
+        if self._keepalive_job is not None:
+            schedule.cancel_job(self._keepalive_job)
+
+        # This hack allows using the "quit" command, so the bot does not appear as "timed out" in the Ts3 Client & Server log
+        if self.ts_connection is not None:
+            self.ts_connection.COMMAND_SET = set(self.ts_connection.COMMAND_SET)  # creat copy of frozenset
```

Comment:
I am a bit confused by this. What is the purpose? Why is the `quit` command not added directly after initialisation?

Reply:
So add "quit" on init ? sure no problem.